dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallKernelMap.java

```diff
@@ -738,7 +738,6 @@ private static boolean containsIgnoreCase(Collection<String> featureToInstall, S
 
             resolver = new RepositoryResolver(productDefinitions, installedFeatures, Collections.<IFixInfo> emptySet(), repoList);
             resolveResult = resolver.resolveAsSet((Collection<String>) data.get(FEATURES_TO_RESOLVE));
-
```

Comment:
can we call ResolveDirector.resolveAutoFeatures right after the resolveAsSet call?